pwnlib/util/misc.py

```diff
@@ -100,13 +100,13 @@ def size(n, abbrev = 'B', si = False):
 
     return '%.02fP%s' % (n / base, abbrev)
 
-KB = 1024
-MB = 1024 * KB
-GB = 1024 * MB
+KB = 1000
```

Comment:
The actual unit is called kB, but such constants should begin with upper case. @heapcrash WDYT?